bundle/compliance/cis_gcp/rules/cis_2_1/test.rego

```diff
@@ -0,0 +1,56 @@
+package compliance.cis_gcp.rules.cis_2_1
+
+import data.cis_gcp.test_data
+import data.compliance.policy.gcp.data_adapter
+import data.lib.test
+
+type := "project-management"
+
+subtype := "gcp-cloudresourcemanager-project"
+
+test_violation {
```

Comment:
TODO: add a test case for multiple `auditLogConfigs ` entries where 1 member has both read/write and the other doesnt